monetio/models/raqms.py

```diff
@@ -30,17 +32,37 @@ def open_dataset(fname, *, surf_only=False):
     ds = xr.open_dataset(names[0], drop_variables=["theta"])
     ds = _fix(ds, surf_only=surf_only)
 
+    # convert all gas species to ppbv
+    if convert_to_ppb:
+        for i in ds.variables:
+            if "units" in ds[i].attrs:
+                if "ppv" in ds[i].attrs["units"]:
```

Comment:
```suggestion
                if ds[i].attrs["units"].startswith("ppv"):
```
a little more specific? (assuming I'm correct it would be in the start) or if it's always just 'ppv' you could use `== "ppv"` instead.

Reply:
I was following the formatting used by the other readers. Switching to `== "ppv"`.